Homework4/ParseTree/ParseTree.Tests/ParseTreeTest.cs

```diff
@@ -0,0 +1,47 @@
+namespace ParseTree.Tests;
+
+using NUnit.Framework;
+using System;
+using ParseTree;
+
+
+public class Tests
+{
+    [SetUp]
+    public void Setup()
+    {
+    }
+
+    private double CreateTreeAndCalculateExpression(string expression)
+    {
+        Tree parseTree = new Tree(expression);
+        return parseTree.Calculate();
+    }
+
+    [TestCase("+ 7 5", ExpectedResult = 12)]
+    [TestCase("- -15 5", ExpectedResult = -20)]
+    [TestCase("* 19 -4", ExpectedResult = -76)]
+    [TestCase("/ 115 4", ExpectedResult = 28.75)]
+    [TestCase("(* 4 (+ (/ 18 -3) (- (* (- 7 -3) 12) 50)))", ExpectedResult = 256)]
+    [TestCase("+ (/ 13 (- 5 3) (/ (+ 1 1) (+ 4 1))", ExpectedResult = 6.9)]
```

Comment:
https://stackoverflow.com/questions/26890837/how-to-add-a-result-tolerance-to-a-nunit-testcase